### DISCUSSION TITLE
If We Get a Logo, We Get a Slogan!

### DISCUSSION THREAD

**Discussion by xee5ch** [oscal-club/community#19] [General]
I have some ideas about a slogan for the work we wish to focus on. I can think of a few communities that have a simple slogan that is kind of like an ethos for the work they do and says something about how they operate.

- The Perl Community has [TIMTOWTDI, There Is More Than One Way To Do It](http://wiki.c2.com/?ThereIsMoreThanOneWayToDoIt).
- The Ruby Community has [MINISWAN, Matz Is NIce So We Are Nice](https://en.wikipedia.org/wiki/Yukihiro_Matsumoto).

I would like to use a cute, ironic slogn I have been using in my work areas:

> WUWDHND: World Unification == World Unification, Have a Nice Day!

The notion here is that, in our community, embracing OSCAL and the compliance and security ecosystem around it is meant to unify regardless of what particular tools and processes and people you bring to the community. The only way validate the hypothesis of OSCAL, Supporting Control-Based Risk Management with Standardized Formats, is to show the maximal extent of unification with the ecosystem is possible. And when we do that, we do it with a little cheer and a humility, so we tell everyone involved to have a nice day. We might disagree or differ, but a nice greeting never hurt anyone.

I want to hear what others think or have alternative to proffer. If I do not hear back from the first members of the community, I intend to stick with this one!

**Comment by jheadley** [comment]
I like the idea. I have no suggestions however. Helpful as always, I know.
